setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "K3s"
+description: Set up K3s on an Ubuntu machine to deploy Coder.
+---
+
+This article will show you how to install K3s onto a new Ubuntu 20.04 LTS
+machine for use with Coder.
+
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that works well
+for single-node or multi-node clusters. This guide covers the installation
+of K3s onto a new Ubuntu 20.04 LTS machine, but if you want to install Coder on
+a local machine or an existing host, [Kind](./kind.md) or [k3d](https://k3d.io/)
+are better choices.
+
+> This installation method is not officially supported or tested by Coder. If
+> you have questions or run into issues, feel free to reach out using our
+> [community Slack channel](https://cdr.co/join-community).
+>
+> **We do not recommend using K3s for production deployments of Coder.**
+
+## Prerequisites
+
+Before proceeding, please make sure that:
+
+1. You have an **Ubuntu 20.04 machine**: This can be a bare metal or a virtual
+   machine. AWS EC2, GCP Compute Engine, Azure VMs, Vultr,
```

Comment:
Is the second sentence referencing the various clouds really necessary here? Seems like as long as it's Ubuntu 20.04, everything should be good, right?

Reply:
Yeah, this can be removed.